sqltest/testdata/activate_source_on_create_mv_test_out.txt

```diff
@@ -0,0 +1,85 @@
+--create topic testtopic;
+use test;
+0 rows returned
+create source test_source_1(
+    col0 bigint,
+    col1 tinyint,
+    col2 int,
+    col3 double,
+    col4 decimal(10, 2),
+    col5 varchar,
+    col6 timestamp,
+    primary key (col0)
+) with (
+    ingestonfirstmv = true,
+    brokername = "testbroker",
+    topicname = "testtopic",
+    headerencoding = "json",
+    keyencoding = "json",
+    valueencoding = "json",
+    columnselectors = (
+        meta("key").k0,
+        v1,
+        v2,
+        v3,
+        v4,
+        v5,
+        v6
+    )
+);
+0 rows returned
+
+--load data dataset_1;
```

Comment:
Add no wait here.
Then add --pause 1000; or something like that after

Reply:
Done